test/docker/Dockerfile

```diff
@@ -1,10 +1,18 @@
 FROM ubuntu:18.04
 
 RUN apt-get update
-RUN apt-get -y install vim iptables build-essential wget git iputils-ping net-tools curl jq kafkacat apt-transport-https socat
+RUN apt-get -y install vim iptables build-essential wget git iputils-ping net-tools curl jq kafkacat apt-transport-https socat software-properties-common lsb-release
 RUN curl -fsSL get.docker.com | sh
 RUN curl https://dl.google.com/go/go1.14.linux-amd64.tar.gz | tar -xzf- -C /usr/local/
 
+# Installing the vault cli (Might not be required since Http API is available anyways)
+ARG HZN_VAULT
```

Comment:
Always install the vault CLI, even if vault isnt started. It doesnt hurt to do it even when vault is absent.